levackt/scrt-swap
# WIP: Feature/create swap

Done/testing
* implement token swap with cli (WIP while testing edge cases and may still be replaced by rest)
* bugfix in operator user config
* configurable fromAccount, can probably be the same as operator user if that's a keyname
* configurable keyring-backend for testing
* configurable chainClient to test on kamut


TODOs
1.  verify unsigned tx was inserted by leader
2. check threshold and broadcast tx
3. verify tx was successful (query mintTransactionHash)
4. update swap status if successful
5. retry failed swaps?
